app/models.py

```
@@ -1,9 +1,9 @@
 """Data models for the Speaking Meeting Bot API."""
 
 from datetime import datetime
-from typing import Any, Dict, List, Optional
+from typing import Any, Dict, List, Literal, Optional
```

Ruff (0.15.20)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)